### ISSUE TITLE
Update CA1712: Add configuration for the number of values to trigger the rule

### ISSUE THREAD

**Issue by Evangelink** [MicrosoftDocs/visualstudio-docs#4381]
I have updated the rule (see https://github.com/dotnet/roslyn-analyzers/pull/3060) to add some configuration. The documentation page shall be updated accordingly.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: dd77481e-7725-7fd9-5010-5194a4fbaf4d
* Version Independent ID: aedf720e-61f0-714a-5815-0d76713738b9
* Content: [CA1712: Do not prefix enum values with type name - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1712?view=vs-2019)
* Content Source: [docs/code-quality/ca1712.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/code-quality/ca1712.md)
* Product: **visual-studio-windows**
* Technology: **vs-ide-code-analysis**
* GitHub Login: @jillre
* Microsoft Alias: **jillfra**

**Comment by Mikejo5000** [comment]
@Evangelink Are you making the doc update? If you'd like me to fix the doc, can you clarify the change that is needed. It's not real clear to me from the link. Thanks!

**Comment by Evangelink** [comment]
@Mikejo5000 I'll let you do it if that's ok. The idea is to add a new `Configurability` section as done for example in https://docs.microsoft.com/en-us/visualstudio/code-quality/ca2007?view=vs-2019#configurability

In this section, you will need to include what is stated in https://github.com/dotnet/roslyn-analyzers/blob/master/docs/Analyzer%20Configuration.md#do-not-prefix-enum-values-with-type-name
Basically, the default behavior is the same but 2 new options were added for this rule in order to change the threshold required for the rule to raise the diagnostic.

Let me know if I wasn't clear.

**Comment by Mikejo5000** [comment]
Thanks! Updating the doc.
